strings.js

```diff
@@ -0,0 +1,51 @@
+// 1. Uppercase the first character
+
+function ucFirst(str) {
+    str = str[0].toUpperCase() + str.slice(1);
+    return str
+}
+
+let text = 'text';
+console.log(ucFirst(text));
+
+// 2. Check for spam
+
+function checkSpam(str, checkStr1, checkStr2) {
+    str = str.toLowerCase();
+    checkStr1 = checkStr1.toLowerCase();
+    checkStr2 = checkStr2.toLowerCase();
+
+    if(str.includes(checkStr1) || str.includes(checkStr2)){
+        return true
+    }
+
+    return false
+}
+
+console.log(checkSpam("asdsd sdf viagra", 'viaGra', 'XXX'));
+
+// 3. Truncate the text
+
+function truncate(str, lengthStr) {
+    return (str.length > lengthStr) ? str.slice(0, lengthStr - 3) + "..." : str
```

Comment:
use template literals instead of `+ "..."`

Reply:
done